after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
```

Comment:
If you remove these lines then the target version variable won't load and the android#isAndroidProject() method will return 0. This means the code inside that condition never executes when you open an XML file and the xml data files will never load.